### ISSUE TITLE
Unsupported ONNX ops of type: Identity

### ISSUE THREAD

**Issue by leonzfa** [onnx/onnxmltools#298]
I used the example code to generate onnx model from keras, and then I convert onnx to mlmodel. It raised the error "Unsupported ONNX ops of type: Identity"

**Comment by vinitra** [comment]
Hi @leonzfa! Thanks for trying onnxmltools. Did you use [onnx-coreml](https://github.com/onnx/onnx-coreml) to convert your ONNX model to the CoreML format? Unfortunately it looks like Identity is not on the [list of supported ops](https://github.com/onnx/onnx-coreml#operators) from the onnx->coreml converter, but can be implemented using a CoreML custom layer.

The developers are very active, so I recommend filing an issue in that repository!

**Comment by leonzfa** [comment]
@vinitra Yes. I used the onnx-coreml for my conversion. But in fact there is no Identity layer in the example code (Keras to ONNX Conversion). Why this onnxmltools generates Identity layer? Maybe it is not neccesary.

**Comment by vinitra** [comment]
@leonzfa, which version of onnx and onnxmltools are you using?

**Comment by Zrufy** [comment]
any news about this layer?I have the same problem.

**Comment by xadupre** [comment]
I wrote a PR to remove unnecessary Identity node in a graph. https://github.com/microsoft/onnxconverter-common/pull/21, we may modify the convert function to use it. It does not solve the inner issue but could unblock you.
